packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -415,14 +398,10 @@ export default class InteractSubmode extends Component {
   private close = (item: StackItem) => {
     // close the item first so user doesn't have to wait for the save to complete
     this.operatorModeStateService.trimItemsFromStack(item);
-    let { request, url } = item;
-
-    // only save when closing a stack item in edit mode. there should be no unsaved
-    // changes in isolated mode because they were saved when user toggled between
-    // edit and isolated formats
-    if (url && item.format === 'edit') {
-      this.store.save(url);
```

Comment:
This automatic save on close is just not necessary. the store will make sure that the save is issued if necessary. this will lighten the indexing load.